omnigibson/examples/object_states/temperature_demo.py

```diff
@@ -133,7 +133,7 @@ def main(random_selection=False, headless=False, short_exec=False):
     for apple in apples:
         apple.states[object_states.Temperature].set_value(-50)
     apples[0].states[object_states.Inside].set_value(oven, True, use_ray_casting_method=True)
-    apples[1].set_position(stove.links["heat_source_link"].get_position() + np.array([0, 0, 0.1]))
+    apples[1].set_position(stove.links["heatsource_link"].get_position() + np.array([0, 0, 0.1]))
     apples[2].states[object_states.OnTop].set_value(tray, True, use_ray_casting_method=True)
```

Comment:
Not necessary any more due to my OnFire PR.

Reply:
Gotcha! I can leave this for now though right? I can address this during the impending merge conflict aftewards